webapp/src/store/language.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSlice, createAsyncThunk, PayloadAction} from '@reduxjs/toolkit'
+
+import {getCurrentLanguage, storeLanguage as i18nStoreLanguage} from '../i18n'
+
+import {RootState} from './index'
+
+export const fetchLanguage = createAsyncThunk(
+    'language/fetch',
+    async () => getCurrentLanguage(),
+)
+
+export const storeLanguage = createAsyncThunk(
+    'language/store',
+    async (lang: string) => {
```

Comment:
This function doesn't seem to need `async`:

```suggestion
    (lang: string) => {
```